packages/serializer/addon/rest.js

```diff
@@ -273,7 +273,7 @@ const RESTSerializer = JSONSerializer.extend({
         continue;
       }
 
-      var isPrimary = !forcedSecondary && this.isPrimaryType(store, typeName, primaryModelClass);
+      var isPrimary = !forcedSecondary && this.isPrimaryType(modelName, PrimaryModelClass);
```

Comment:
if we capitalize the argument here we need to capitalize it elsewhere in this method as well (this is why the basic tests are failing)

Reply:
Of course haha 😅 my bad 

Will have a look at new failing ones